src/App/routes/Channel/ChannelData.tsx

```diff
@@ -30,7 +30,7 @@ export function ChannelData({ portId, channelId }: ChannelDataProps): JSX.Elemen
       {portId ? <div>Port ID: {portId}</div> : null}
       {channelId ? <div>Channel ID: {channelId}</div> : null}
       <div>Proof: {channelResponse.proof?.length ? toHex(channelResponse.proof) : "–"}</div>
-      <HeightData height={channelResponse.proofHeight} />
+      <HeightData height={channelResponse.proofHeight ?? null} />
```

Comment:
Should these use a different placeholder? Eg `"-"`.

Reply:
Hmm, I added this because my IDE complained about it, but the compiler seems fine with it.

Reply:
I was referring to your comment about the fallback, not to the diff. `HeightData` is taking care of the null case any sets some placeholder.

Reply:
Ah, well I still don't know why my IDE is out of sync with tsc. I'm guessing something to do with React scripts.